core/src/program/blocks/span_block.rs

```diff
@@ -52,7 +54,27 @@ impl Span {
     pub fn new(operations: Vec<Operation>) -> Self {
         assert!(!operations.is_empty()); // TODO: return error
         let (op_batches, hash) = batch_ops(operations);
-        Self { op_batches, hash }
+        Self {
+            op_batches,
+            hash,
+            decorators: DecoratorList::new(),
+        }
```

Comment:
Here, you can probably just call `with_decorators()` and pass an empty vector as the second parameter.

Reply:
Thanks. Done.